README.md

```diff
@@ -6,79 +6,104 @@
+> check against this repository. This is a mandatory prerequisite before being
+> added to the registy. 
+
 ### 0. Fork this repository
 You will be raising a Pull Request from your fork to the upstream repo.
 
-### 1. Ensure your chain is listed at ethereum-lists/chains
-This is to ensure your chain has a unique chain ID. Our validation suite will check your chain against https://github.com/ethereum-lists/chains.
+### 1. Install dependencies
 
+Install the following dependencies
 
-### 2. Install dependencies
-You will need [`jq`](https://jqlang.github.io/jq/download/) and [`foundry`](https://book.getfoundry.sh/getting-started/installation) installed, as well as Go and [`just`](https://just.systems/man/en/).
+| Dependency                                                            | Version   | Version Check Command |
+| --------------------------------------------------------------------- | --------- | --------------------- |
+| [git](https://git-scm.com/)                                           | `^2`      | `git --version`       |
+| [go](https://go.dev/)                                                 | `^1.21`   | `go version`          |
+| [foundry](https://github.com/foundry-rs/foundry#installation)         | `^0.2.0`  | `forge --version`     |
+| [jq](https://github.com/jqlang/jq)                                    | `^1.6`    | `jq --version`        |
+| [just](https://github.com/casey/just?tab=readme-ov-file#installation) | `^1.28.0` | `just --version`      |
 
-### 3. Set env vars
+### 2. Set environment variables
 
-To contribute a standard OP-Stack chain configuration, in addition to user-supplied metadata (chain name) the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local [monorepo](https://github.com/ethereum-optimism/optimism) folder.
-
-First, make a copy of `.env.example` named `.env`, and alter the variables to appropriate values. Each value is explained in a comment in `.env.example`.
+Make a copy of `.env.example` named `.env`, and alter the variables to appropriate values. Each value is explained in a comment in `.env.example`.
 
 #### Frontier chains
 To contribute a frontier OP-Stack chain configuration, you set the `SCR_CHAIN_TYPE=frontier` in the `.env` file. This will set `superchain_level = 1` in the registry's config file for this chain.
 
 #### Standard chains
+
+todo: is this different from frontier chains? and which tool are we referring to?
+To contribute a standard OP-Stack chain configuration, in addition to user-supplied metadata (chain name) the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local [monorepo](https://github.com/ethereum-optimism/optimism) folder.
+
 Adding a standard chain is a two-step process. First, the chain should be added as a frontier chain as above, but with `SCR_STANDARD_CHAIN_CANDIDATE=true` in the `.env` file.
```

Comment:
This is actually kind of weird, that you have to set `SCR_CHAIN_TYPE=frontier` for standard chains. Presumably these get turned into numerical levels on the other end, but it's weird that we ask people to type out `frontier` for their standard candidate chains